js/stores/appStore.js

```diff
@@ -235,14 +235,20 @@ const createWindow = (action) => {
     mainWindow.webContents.on('did-finish-load', (e) => {
       lastEmittedState = appState
       mainWindow.webContents.setZoomLevel(zoomLevel[toolbarUserInterfaceScale] || 0.0)
-      e.sender.send(messages.INITIALIZE_WINDOW,
-        {
+
+      let state = appState.toJS()
+      delete state.menu
```

Comment:
This won't work because Windows requires it when rendering the custom menubar (in the renderer process).  I'll look at submitting a PR which avoids setting it on macOS / Linux and properly sets it (ex: doesn't bind raw object with functions) on Windows